vipps-ecom-api.md

```diff
@@ -691,6 +692,26 @@ This is useful when a customer does the following:
 4. Adds another product to the same cart (or order)
 5. Repeats steps 2 and 3.
 
+### transactionText recommendations
+
+All Vipps payments are initiated with a text that appears in Vipps, using the
+`transactionText` parameter. There are separate texts for the different actions:
+Initiate, capture, cancel, refund.
+
+The text should be as user-friendly and informative as possible.
+
+The `transactionText` can be up to 100 characters.
+
+We recommend the following, using the example "Vipps webshop":
+
+| Operation         | Recommended `transactionText`     | Not recommended |
+| ----------------- | --------------------------------- | -------------------- |
+| Initiate, one item (the `transactionText` is shown in the push message and on the payment screen)| `One pair of Vipps socks`, `Orderid acme-shop-123-order123abc in Vipps webshop`        | `Payment`, `Vipps webshop`, `Shop number 12345`, `Vipps` |
+| Initiate, multiple items or a shopping basket | `One pair of Vipps socks, one XL hoodie, ...` (truncated to ~100 characters) or `Payment for order acme-shop-123-order123abc in Vipps webshop` | (same as above) |
+| Initiate, reservation of maximum amount (for EV charging, parking, etc) | `Midlertidig reservasjon. Du bli kun belastet for faktisk bruk.` English: `Temporary reservation. You will only be charge for the actual use.` | Not making it clear that it is a temporary reservation of the maximum amount. (otherwise: same as above) |
```

Comment:
```suggestion
| Initiate, reservation of maximum amount (for EV charging, parking, etc) | `Midlertidig reservasjon. Du bli kun belastet for faktisk bruk.` English: `Temporary reservation. You will only be charged for the actual use.` | Not making it clear that it is a temporary reservation of the maximum amount. (otherwise: same as above) |
```